src/utils.c

```diff
@@ -766,21 +766,47 @@ gchar *utils_get_date_time(const gchar *format, time_t *time_to_use)
 }
 
 
+/* Extracts initials from @p name, with some Unicode support */
+GEANY_EXPORT_SYMBOL
 gchar *utils_get_initials(const gchar *name)
 {
-	gint i = 1, j = 1;
-	gchar *initials = g_malloc0(5);
+	GString *initials;
+	gchar *composed;
+	gboolean at_bound = TRUE;
+	gboolean prev_matched = FALSE;
+
+	g_return_val_if_fail(name != NULL, NULL);
+
+	composed = g_utf8_normalize(name, -1, G_NORMALIZE_DEFAULT_COMPOSE);
```

Comment:
I suspect it should be `G_NORMALIZE_ALL_COMPOSE` so composite characters get split and other compatibility transforms. (==NFKC?).

I am unsure if (for example) ligatures can occur as first letters of names, and if they do is the "initial" the ligature or the first letter of it?   Currently it selects the ligature as the "initial".  Anyway its not critical so long as the user can override it with the "initials" preference.

Reply:
If you have more knowledge on combining, I'll take it, I only know accents myself :) (and then some ligatures that have assigned characters and that I feel like being variants, like Œ or Æ)

Reply:
Oooh, Æsop, there is a name with a ligature, so I guess its possible.

My rule is that if its inconvenient and annoying and inconsistent some natural language will do it!!!